datapacks/Bookshelf/data/bs.block/function/__load__.mcfunction

```diff
@@ -15,6 +15,9 @@
 # Documentation of the feature: https://bookshelf.docs.gunivers.net/en/latest/modules/block.html
 # ------------------------------------------------------------------------------------------------------------
 
+forceload add -30000000 1600
+setblock -30000000 0 1606 minecraft:decorated_pot
```

Comment:
Do we have to document this new block in [shared blocks](https://mc-bookshelf--222.org.readthedocs.build/en/222/contribute/shared-data.html#shared-blocks)?

Reply:
yeah i think so !